src/OrbitGl/FunctionsDataView.cpp

```diff
@@ -108,23 +108,8 @@ std::string FunctionsDataView::GetValue(int row, int column) {
       return absl::StrFormat("%lu", function.size());
     case kColumnModule:
       return orbit_client_data::function_utils::GetLoadedModuleName(function);
-    case kColumnAddress: {
-      const ProcessData* process = app_->GetTargetProcess();
-      // If no process is selected, that means Orbit is in a disconnected state aka displaying a
-      // capture that has been loaded from file. CaptureData then holds the process
-      if (process == nullptr) {
-        const CaptureData& capture_data = app_->GetCaptureData();
-        process = capture_data.process();
-        CHECK(!app_->IsCaptureConnected(capture_data));
-      }
-      CHECK(process != nullptr);
-      const ModuleData* module =
-          app_->GetModuleByPathAndBuildId(function.module_path(), function.module_build_id());
-      CHECK(module != nullptr);
-      return absl::StrFormat("0x%llx", orbit_client_data::function_utils::GetAbsoluteAddress(
-                                           function, *process, *module)
-                                           .value_or(0));
-    }
+    case kColumnAddress:
+      return absl::StrFormat("%#" PRIx64, function.address());
```

Comment:
So we are only changing the address in those data view to be the relative one and not in the disassembly? Which addresses are displayed in the sampling view? We should take care that we remain somehow consistent. E.g. if I have a callstack sample and go to disassembly, I should be able to immediately correlate that sample with the instruction. I think this is still the case, but just wanted to check back.

Reply:
Sampling view uses absolute addresses. But here we do not have them because function list in this view is coming from local modules not server-side mapping so it is fine to have elf-addresses here. Note also that sort currently uses relative addresses which was not consistent with getvalue.

Reply:
Makes sense. Thanks!